tests/duration.rs

```diff
@@ -577,3 +577,21 @@ fn test_min_max() {
     assert_eq!(d1, d1.max(d0));
     assert_eq!(d1, d0.max(d1));
 }
+
+#[test]
+fn regression_test_gh_244() {
+    use hifitime::Epoch;
+
+    let _local0 = if let Ok(x) = Epoch::maybe_from_gregorian_utc(1792, 1, 1, 0, 0, 0, 0) {
```

Comment:
Let's call `unwrap` here directly so that if this initialization fails for whatever reason, the test will fail directly.

```suggestion
    let _local0 = Epoch::maybe_from_gregorian_utc(1792, 1, 1, 0, 0, 0, 0).unwrap();
```